src/main/kotlin/dev/fakek/fakes/FakeCompany.kt

```diff
@@ -0,0 +1,39 @@
+package dev.fakek.fakes
+
+/**
+ * FakeCompany provides a company with random information.
+ *
+ * @param bs company bs.
```

Comment:
I think the rest of the params are self explanatory, would it be possible to include a few samples of what `bs` maps to? If I remember correctly it was corporate speak but I think a few samples would be helpful.

Reply:
check it now and see if it is enough i took it from some random BS generator as example, but I can change to some random generated text from the java lib if you think it would be better